src/web/utils/ErrorView.tsx

```diff
@@ -0,0 +1,28 @@
+const analyticsIdentifier = (errorId?: string, errorHash?: string) => {
+  if (!errorHash && !errorId) return 'No error identifier';
+  return errorHash
+    ? `error hash: ${errorHash}` // From backend
+    : `error ID: ${errorId}`; // Generated by frontend to make it possible to triage user reported errors.
+};
+
+export function ErrorView({
+  message,
+  errorId,
+  errorHash,
+}: {
+  message?: string;
+  errorId?: string;
+  errorHash?: string;
+}) {
+  return (
+    <div className='error-content'>
+      <img alt='Error icon' src='/uid2-logo.png' />
+      <div>Error</div>
+      <div>
+        {message ??
+          'Unexpected error encountered, please contact support if the problem persists and provide the information below'}
```

Comment:
I added a default message - it is all we ever used anyway. I'm happy to remove the `message` prop entirely if we don't see the need for it.